### ISSUE TITLE
Urql fails with "Generic type 'QueryProps<T, V>' requires 2 type argument(s).  TS2314"

### ISSUE THREAD

**Issue by pnordhus** [dotansimha/graphql-code-generator#2024]
**Describe the bug**
When using the typescript-urql plugin, any autogenerated file with React Components fails compiling with:

```
TypeScript error in /home/user/testapp/src/graphql.tsx(30,15):
Generic type 'QueryProps<T, V>' requires 2 type argument(s).  TS2314

    28 | 
    29 | export const TestComponent = (
  > 30 |   props: Omit<Urql.QueryProps, "query"> & { variables?: TestQueryVariables }
       |               ^
    31 | ) => <Urql.Query {...props} query={TestDocument} />;
    32 | 
```

**To Reproduce**

1. My GraphQL schema:

```graphql
type Query {
	test: String
}
```

2. My GraphQL operations:

```graphql
query test {
	test
}
```

3. My `codegen.yml` config file:

```yml
schema: schema.graphql
generates:
  src/graphql.tsx:
    documents:
      - test.graphql
    plugins:
      - typescript
      - typescript-operations
      - typescript-urql
```

**Expected behavior**

The autogenerated file `src/graphql.tsx` should compile.

**Environment:**

- OS: Arch Linux
- `@graphql-codegen/...`: 1.2.1
- NodeJS: 10.16.0
- Typescript: 3.5.1
- Urql: 1.1.3

**Comment by BjoernRave** [comment]
I experience the same problem

**Comment by dotansimha** [comment]
Fixed in 1.4.0 🎉
